cmd/tools/mockgen/main.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"go.temporal.io/server/tools/mockgen"
+)
+
+func main() {
+	if err := mockgen.Run(os.Args[1:]); err != nil {
```

Comment:
do we really need this level of indirection? can we just put the code in here?

Reply:
I think it's useful because we can't add tests to a `main` package AFAIK

Reply:
fair enough